db/migrate/20160218161746_add_csv_export.rb

```diff
@@ -0,0 +1,10 @@
+class AddCsvExport < ActiveRecord::Migration
+  def change
+    create_table :csv_exports do |t|
+      t.integer :user_id,    null: false
+      t.timestamps           null: false
+      t.text :filters
```

Comment:
Had to change to string to make it work with mysql.  I don't see a case where the filter string will grow to > 255 characters now that the filter is controlled.